src/styles/main.scss

```diff
@@ -1 +1,57 @@
-// put code here
+.body {
+  width: 100%;
+  height: 100vh;
+  display: grid;
+  place-items: center;
+  background-color: black;
+}
+
+.stopwatch {
+  position: relative;
+  width: 500px;
+  height: 500px;
+  border: 15px dotted darkred;
+  border-radius: 50%;
+
+  &__center-button {
+    position: absolute;
+    top: 235px;
+    left: 235px;
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+    background-color: yellow;
+  }
+
+  &__minute-hand {
+    position: absolute;
+    top: 0;
+    left: 245px;
+    height: 250px;
+    width: 10px;
+    background-color: blue;
+    transform-origin: bottom center;
+    animation: rotate 5s infinite steps(60);
+  }
+
+  &__second-hand {
+    position: absolute;
+    top: 100px;
+    left: 240px;
+    height: 150px;
+    width: 20px;
+    background-color: blue;
+    transform-origin: bottom center;
+    animation: rotate 60s infinite steps(60);
+  }
```

Comment:
you have some duplicates here,
create a common class for instance `&__hand`, and set styles by default. and for second-hand override that you need